client/src/components/Banner/Banner.css

```diff
@@ -18,3 +18,21 @@
   line-height: 72px;
   margin-bottom: 5vh;
 }
+.Banner-booktrip{
```

Comment:
`booktrip` is very contextual information. Is there a way to handle this without creating a whole new prop just for this one use case?

Reply:
Most of this CSS is probably unnecessary, right? It looks like banner can already handle having a background image. The only difference here is that the text has to be in the bottom right hand corner

Reply:
@ZakkMan changed from Banner-booktrip to class too Banner--large.